### ISSUE TITLE
Add support and option to connect using Coinbase Wallet

### ISSUE THREAD

**Issue by pabloruiz55** [austintgriffith/scaffold-eth#225]
When clicking "connect" on the top right you should be given the option to use Coinbase wallet in addition to MM and WalletConnect

https://blog.coinbase.com/coinbase-wallet-introduces-new-browser-extension-dd067403b86

**Comment by codenamejason** [comment]
With the increasing use of Coinbase, this is a good idea to have it available to the developers.

**Comment by PowVT** [comment]
I actually have a bit of experience here and could work on a coinbase implementation for the connect options. You have to create a custom Web3 provider using the web3modal docs from what I remember. I can work on it if you would like

**Comment by pabloruiz55** [comment]
Sure, by all means! I'll assign it to you. Let's touch base in a couple weeks maybe?

**Comment by PowVT** [comment]
Hey so I found my notes of when I tried to implemented coinbase previously. I pulled a fresh scaffold-eth master and made changes to allow for Coinbase wallet mobile app based on my notes. If you guys don't mind giving it a test for yourself and let me know if anything comes up that would be awesome. Here is the repo: https://github.com/PowVT/scaffold-eth. Don't forget to change the in-app settings to be pointed at same chain your app is deployed to!
